EliteDangerous/JournalEvents/JournalBountyCrimesBondsFines.cs

```diff
@@ -460,17 +460,19 @@ public JournalClearImpound(JObject evt) : base(evt, JournalTypeEnum.ClearImpound
             ShipId = evt["ShipID"].ULong();
             MarketID = evt["MarketID"].Long();
             ShipMarketID = evt["ShipMarketID"].Long();
+            System = evt["System"].StrNull();
         }
 
         public string ShipType { get; set; }
         public string ShipType_Localised { get; set; }
         public ulong ShipId { get; set; }
         public long ShipMarketID { get; set; }
         public long MarketID { get; set; }
+        public string System { get; set; }  //patch 17, so may be null
 
         public override void FillInformation(out string info, out string detailed)
         {
-            info = BaseUtils.FieldBuilder.Build("Ship: ".T(EDCTx.JournalEntry_Ship), ShipType_Localised);
+            info = BaseUtils.FieldBuilder.Build("Ship: ".T(EDCTx.JournalEntry_Ship), ShipType_Localised, "In system: ".T(EDCTx.JournalClearImpound_InSystem), System);
```

Comment:
Missing ID

Reply:
You can see it was in there in the first commit: https://github.com/EDDiscovery/EliteDangerousCore/pull/133/commits/9e4ffe9d10f7d4c2feaddfda10d1d0f2e38d6fdb
You can steal it back from there.